uktrade/great-international-ui
# Display intro video on investment opportunity page

This PR displays the new `intro_video` field on an investment opportunity page. If there is no intro-video, it will display the intro_image.

It also includes a styling change which makes the scrollbar on the video transcript box always visible, making it clearer to the user that the text box is scrollable.

**Screenshot**
<img width="1290" alt="Screenshot 2022-11-17 at 09 55 56" src="https://user-images.githubusercontent.com/22460823/202415113-99375350-5750-47bb-8a96-d046fe3c8b11.png">


**To test**
* Pull latest from directory-cms
* Upload an intro_video to this opportunity http://cms.trade.great:8010/admin/pages/786/
* Go to the [corresponding page in international-ui](http://international.trade.great:8012/international/content/investment/opportunities/sustainable-aquaculture-in-dorset/) and check that the video appears as expected

 - [x] Change has a jira ticket that has the correct status.
 - [x] (if adding CSS) CSS have been compiled.
 - [x] (if adding HTML) Change is accessible to the standards we subscribe to.
 - [x] (if changing the UI) Add a printscreen to the PR
